styles.css

```diff
@@ -339,3 +347,126 @@
     color: #fff;
   }
 }
+
+/* @media screen and (min-width:  768px) { */
+@media screen and (min-width: 768px) {
+  .info-image {
+    display: none;
+  }
+
+  .main-header {
+    position: fixed;
+    width: 100%;
+    box-shadow: rgba(0, 0, 0, 0.1) 0 4px 12px;
+    z-index: 6;
+  }
+
+  .menu-lists {
+    display: flex;
+    list-style-type: none;
+  }
+
+  .menu-lists .item a {
+    text-transform: capitalize;
+    font-size: 15px;
+    color: #344563;
+    margin-right: 32px;
+    cursor: pointer;
+  }
+
+  .info {
+    max-width: 1200px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: auto;
+  }
+
+  .headlines {
+    background-image: url("assets/images/header-bg.svg");
+    background-position: center;
+    background-size: contain;
+    background-repeat: no-repeat;
+    min-height: 80vh;
+    position: relative;
+    top: 50px;
+    margin-bottom: 60px;
+  }
+
+  .contain {
+    max-width: 1000px;
+    padding: 24px;
+    font-family: "Poppins", sans-serif;
+
+    /* margin: auto; */
```

Comment:
- Kindly remove this commented-out line of code to make your code clean :recycle: